Homework/roman/DecimalBinaryExchanger/Program.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace DecimalBinaryExchanger
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("Enter a number: ");
+            int number = Int32.Parse(Console.ReadLine());
+            string binary = Convert.ToString(number, 2);

```

Comment:
this program should use a loop to convert values both from binary to decimal and vice versa!